### ISSUE TITLE
[Electron Vue] Failed to mount component: template or render function not defined.

### ISSUE THREAD

**Issue by dalgopower** [MisRob/vue-tree-navigation#39]
Hi

I am developing a desktop application based on Electron-vue and I am trying to use VueTreeNavigation on my app, however cannot solve this problem.

I see this warning message on Console of Electron application window.

> [Vue warn]: Failed to mount component: template or render function not defined.

I have searched a lot on the internet to find a solution, but no success. And I even copied the source itself to my project and tried to run with it and again no success.

I saw some people saying that Vue project/modules for pure(?)/web are not suitable with Electron applications.

Can anyone give me a solution with this problem? or At least tell me a reason why it does not work?

Thank you

**Comment by marcuus** [comment]
Yes I get the same message. I dont think you can use this as a component.
